kafka-ui-react-app/src/components/ConsumerGroups/List/__test__/ListItem.spec.tsx

```diff
@@ -54,50 +54,55 @@ describe('List', () => {
   );
 
   it('render empty ListItem', () => {
-    expect(component.exists('tr')).toBeTruthy();
+    expect(component.getByRole('row')).toBeInTheDocument();
   });
 
   it('renders item with stable status', () => {
-    const wrapper = mount(
+    render(
       setupWrapper({
         ...mockConsumerGroup,
         state: ConsumerGroupState.STABLE,
       })
     );
-
-    expect(wrapper.find('td').at(5).text()).toBe(ConsumerGroupState.STABLE);
+    expect(screen.getByRole('row')).toHaveTextContent(
+      ConsumerGroupState.STABLE
+    );
   });
 
   it('renders item with dead status', () => {
-    const wrapper = mount(
+    render(
       setupWrapper({
         ...mockConsumerGroup,
         state: ConsumerGroupState.DEAD,
       })
     );
 
-    expect(wrapper.find('td').at(5).text()).toBe(ConsumerGroupState.DEAD);
+    expect(screen.queryAllByRole('cell')[5]).toHaveTextContent(
```

Comment:
replace query by getAll -> since it is in the document , use query when the text or the item is not in the doc

Reply:
done